### PR TITLE
[API] Adding files in the docker container for API integration tests 

### PR DESCRIPTION
**Pull request by spell00**
## Brief summary of changes

The integration test for the GET request to download an image had to be skipped because the docker container don't have the data in `/data-raisinbread`.

The data is not available during Travis checks, so empty files are added and, during Travis checks, are moved to where the data should be to run the integration tests that assert the files can be downloaded with the API.

#### Link(s) to related issue(s)

* Resolves #6907

### CODE REVIEW

**Comment by spell00** [comment]
The changes I propose in this PR don't work with Travis because the raisinbread data is not included. The only solution I could see would be to have the data for these endpoints stored somewhere where they can be downloaded. The instructions to download them could be encrypted as here: https://docs.travis-ci.com/user/encrypting-files/ and added in the git repo. It would then be decrypted when Travis is run, then the instructions it contains would be used to download the secured data.

File: docker-compose.yml
Comment (kongtiaowang) [inline docker-compose.yml:11]: this line is not necessary.

File: docker-compose.yml
Comment (kongtiaowang) [inline docker-compose.yml:20]: This line is not necessary.

File: docker-compose.yml
Comment (kongtiaowang) [inline docker-compose.yml:59]: This line is not necessary.

**Review by kongtiaowang — CHANGES_REQUESTED** [review]

File: test/unittests/BreadcrumbTrailTest.php
Comment (kongtiaowang) [inline test/unittests/BreadcrumbTrailTest.php:47]: These files have confilcts, you need to fix them.

**Review by kongtiaowang — CHANGES_REQUESTED** [review]

File: docker-compose.yml
Comment (spell00) [reply docker-compose.yml:11]: I did try to remove it (and the others you marked as not necessary), but then it didn't work anymore. Unfortunately, I am unable to test this change right now. As I think you may have noticed, there is a problem with `composer` packages that makes Travis unable to run checks (ends up being errored). This is also causing errors when I try to run tests today...

I tried to add the ` composer/package-versions-deprecated` package as suggested and it did solve the composer packages problem, but it shows a lot of new valid formatting problems that were not detected before.

Comment (kongtiaowang) [reply docker-compose.yml:11]: I will fix the composer issue. Then you can test it again.

Comment (spell00) [reply docker-compose.yml]: I was able to run the tests and it looks like you were right, it works without the 3 lines you said were not necessary. I removed them.

File: docker-compose.yml
Comment (spell00) [reply docker-compose.yml]: I removed it

File: docker-compose.yml
Comment (spell00) [reply docker-compose.yml]: It is removed

File: test/unittests/BreadcrumbTrailTest.php
Comment (spell00) [reply test/unittests/BreadcrumbTrailTest.php]: it is resolved.

**Comment by christinerogers** [comment]
@kongtiaowang it looks like @spell00 has addressed your comments.  
Could you re-review when you have a moment friday?

**Review by kongtiaowang — APPROVED** [review]
LGTM , This PR will be rebase after fixing Travis.
https://github.com/aces/Loris/pull/7101

**Comment by christinerogers** [comment]
@driusan requesting final review - this is our last stretch in which @spell00 can readily make changes.  thank you!

**Comment by spell00** [comment]
blocked by #7101

**Comment by spell00** [comment]
@driusan @xlecours This is now passing Travis checks and is ready for final review

**Review by driusan — APPROVED** [review]
Since they're empty I don't see why these files are added to raisinbread instead of just created by the test suite's setup if they don't exist, but I don't think it's a blocker so I'll merge it anyways.

**Pull request merged**
